pkgs/by-name/fi/finalcut/package.nix

```
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoreconfHook,
+  autoconf-archive,
+  ncurses,
+  pkg-config,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "finalcut";
+  version = "0.9.1";
+
+  src = fetchFromGitHub {
+    owner = "gansm";
+    repo = "finalcut";
+    tag = finalAttrs.version;
+    hash = "sha256-iKLE4UMDbnsKYEjQHlF+xyZSBke1EZSVJiabbKRkzhg=";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    autoconf-archive
+    pkg-config
+  ];
+
+  buildInputs = [ ncurses ];
+
+  meta = {
+    description = "A text-based widget toolkit";
+    homepage = "https://github.com/gansm/finalcut";
+    license = lib.licenses.lgpl3Plus;
```

```suggestion
    license = lib.licenses.lgpl3Only;
```

maybe finalcut is granted the LGPLv3 only license. [ref](https://github.com/gansm/finalcut#license)

---

But here https://github.com/gansm/finalcut/blob/c294f2857fed6b0c63b1a3d157bbc66352e9fbd4/final/final.h#L9-L12:
```
* FINAL CUT is free software; you can redistribute it and/or modify    *
* it under the terms of the GNU Lesser General Public License as       *
* published by the Free Software Foundation; either version 3 of       *
* the License, or (at your option) any later version.                  *
```

---

@sikmir I overlooked this comment, sorry about that.